<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="navigationBG" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#navigationBG {
background-color: #ccdde8;

}

</string>
    </property>
    <widget class="QLabel" name="systemLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>15</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#systemLabel {
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: #034488;
}</string>
     </property>
     <property name="text">
      <string>Movie
Ticketing
System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>680</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/logout.png</normaloff>../img/logout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="movieButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Movies</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/movies.png</normaloff>../img/movies.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="movieButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bookings</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/tickets.png</normaloff>../img/tickets.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainBG" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>731</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainBG {
background-color: #fffcf6;

}

</string>
    </property>
    <widget class="QPushButton" name="bookTicket">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#bookTicket{
background-color: #ccdde8;
border: 1px solid #ccdde8;
border-radius: 15px;
}

#bookTicket:hover{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string>Book A Ticket</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>631</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#stackedWidget{
background-color: #178fd6;
border: 1px white;
border-radius: 20px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>631</width>
         <height>601</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
     <widget class="QWidget" name="page_3"/>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>710</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#back{
background-color: rgb(85, 255, 255);
border: 1px solid white;
border-radius: 10px;
}
#back:hover{
background-color: white;
border: 1px solid white;
border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/prev.png</normaloff>../img/prev.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="page_4">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>715</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="rect" native="true">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>704</y>
       <width>161</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#rect {
background-color: rgb(213, 213, 213);
border-radius: 15px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>710</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#next{
background-color: rgb(85, 255, 255);
border: 1px solid white;
border-radius: 10px;
}
#next:hover{
background-color: white;
border: 1px solid white;
border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/next.png</normaloff>../img/next.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../img/moviesHeader.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>rect</zorder>
    <zorder>bookTicket</zorder>
    <zorder>stackedWidget</zorder>
    <zorder>back</zorder>
    <zorder>page_4</zorder>
    <zorder>next</zorder>
    <zorder>label_2</zorder>
   </widget>
   <zorder>mainBG</zorder>
   <zorder>navigationBG</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
